apps/src/applab/designElements/button.jsx

```diff
@@ -200,21 +197,116 @@ class ButtonEvents extends React.Component {
   }
 }
 
+// Initial button size when fontSize is 14 or smaller is 80x30 (classic theme)
+// Initial button size when fontSize is 15 or greater is 100x40 (new themes)
+const MAX_SMALL_FONT_SIZE = 14;
+const DEFAULT_BUTTON_WIDTH = '100px';
+const DEFAULT_BUTTON_WIDTH_SMALL = '80px';
+const DEFAULT_BUTTON_HEIGHT = '40px';
+const DEFAULT_BUTTON_HEIGHT_SMALL = '30px';
+
 export default {
   PropertyTab: ButtonProperties,
   EventTab: ButtonEvents,
+  themeValues: {
+    backgroundColor: {
+      type: 'color',
+      ...themeColor.buttonBackground
+    },
+    borderRadius: {
```

Comment:
These shouldn't live here. It'd be very difficult to add another theme.

Reply:
I did this consolidation for colors, but didn't for the components themselves. There are pros and cons. Each component has different props that can be styled, if they were all the same, it would make a little more sense to pull them out.

FWIW, I have added tests that check that all themes have the relevant values, so I don't think that is a concern.

Reply:
I did see the tests (thanks for adding those!), but I think it this will still be difficult to maintain. If the themes feature grows into something bigger (i.e. student-created style sheets), it will be hard to work with this model.